runtime/vm/jvminit.c

```diff
@@ -4074,6 +4074,41 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		}
 	}
 
+	{
+		IDATA disclaimIndex = FIND_AND_CONSUME_VMARG(STARTSWITH_MATCH, VMOPT_XXCLASSMEMORYDISCLAIM, NULL);
+		if (0 <= disclaimIndex) {
+			PORT_ACCESS_FROM_JAVAVM(vm);
+#if defined(LINUX)
+			char *disclaimOption = NULL;
+			if (4096 != j9vmem_supported_page_sizes()[0]) {
```

Comment:
If the command line option is `-XX:ClassMemoryDisclaim=none` then we should not abort in this case.